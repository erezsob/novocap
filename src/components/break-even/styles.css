.container {
    display: flex;
    flex-direction: row;
    justify-content: space-between;
    align-items: center;
    height: 20px;
    width: 90%;
    padding-top: 20px;
    color: white;
    font-weight: bold;
    font-size: 20px;
}

.barContainer {
    margin: 20px;
    width: 150px;
    height: 150px;
    position: relative;
    display: flex;
    flex-direction: column;
    justify-content: center;
}

.title {
    display: flex;
    flex-direction: column;
    padding-top: 13px;
}

.date {
    font-size: 11px;
    font-weight: 300;
}

.addCosts {
    color: #fff;
    font-size: 19px;
}