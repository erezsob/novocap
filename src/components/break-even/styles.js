import { blueMain } from '../../common/variables.css';

export default {
  card: {
    display: 'flex',
    flexDirection: 'column',
    alignItems: 'center',
    boxShadow: 'none',
    height: 208,
    backgroundColor: blueMain,
    cursor: 'pointer'
  },
  iconButton: {
    color: 'white',
    width: 24
  }
};
