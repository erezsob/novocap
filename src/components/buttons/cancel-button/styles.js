import { pinkyRed } from '../../../common/variables.css';

export default {
  cancel: {
    color: pinkyRed
  }
};
