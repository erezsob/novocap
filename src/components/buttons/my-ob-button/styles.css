@value blueLight, bpXSmall from '../../../common/variables.css';

.button {
    border-radius: 4px;
    background-color: blueLight;
    color: white;
    padding: 8px 0;
    font-size: 12px;
    display: table;
    border: 0;
    outline: 0;
    cursor: pointer;
}

.bigger {
    padding: 10px 0;
    font-size: 16px;
}

.icon,
.text {
    vertical-align: middle;
    text-align: center;
}

.text {
    padding: 0 20px;
    font-weight: lighter;
}

.icon ~ .text {
    display: none;
}

.icon {
    display: table-cell;
    padding: 0 10px;
    position: relative;
    top: 2px;
}

@media (min-width: bpXSmall) and (min-device-width: bpXSmall) {
    .icon ~ .text {
        padding-left: 0;
        display: table-cell;
    }
}
