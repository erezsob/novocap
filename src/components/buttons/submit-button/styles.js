import { greenGrass } from '../../../common/variables.css';

export default {
  submit: {
    color: greenGrass
  }
};
