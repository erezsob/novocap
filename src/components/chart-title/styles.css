.label {
    letter-spacing: 0.05em;
    font-size: 20px;
    border-bottom: 1px solid #efefef;
    padding: 20px;
    margin-bottom: 20px;
    font-weight: bold;
    color: #323435;
}