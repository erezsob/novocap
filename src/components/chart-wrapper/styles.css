.container {
    padding: 0 20px 30px;
    position: relative;
    font-size: 12px;
}