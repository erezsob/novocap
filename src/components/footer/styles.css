@value defaultText, footerBlue, blueDark from './../../common/variables.css';

.footer {
    display: flex;
    flex-direction: column;
    height: 260px;
    width: 100%;
    background-color: footerBlue;
    text-align: center;
    justify-content: center;
    box-sizing: border-box;
    color: defaultText;
}

.legal,
.languages {
    color: blueDark;
}

.legal {
    margin-bottom: 60px;
}

.languages {
    margin-bottom: 20px;
}

.footer .currentLang {
    pointer-events: none;
    color: defaultText;
}

.languages .link,
.company {
    font-size: 12px;
    letter-spacing: 0;
}

.link {
    font-size: 14px;
    letter-spacing: 0.02em;
    padding: 0 5px 0 6px;
    position: relative;
}

.link:after {
    content: '|';
    color: defaultText;
    margin-left: 5px;
    position: absolute;
    font-size: 12px;
    right: -5px;
    top: 3px;
}

.link:last-child:after {
    content: '';
}


