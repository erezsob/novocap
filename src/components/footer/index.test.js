import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import Footer from './';
import styles from './styles.css';

describe('<Footer />', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Footer />);
  });

  it('renders properly', () => {
    const tree = renderer.create(<Footer />).toJSON();
    expect(tree).toMatchSnapshot();
    expect(wrapper).toHaveLength(1);
  });

  it('renders with a class `footer`', () => {
    expect(wrapper.hasClass(`${styles.footer}`)).toBeTruthy();
  });

  it('renders a section with class `legal`', () => {
    expect(wrapper.find(`.${styles.legal}`)).toHaveLength(1);
  });

  it('renders 6 legal links', () => {
    expect(wrapper.find(`.${styles.legal}`).children()).toHaveLength(6);
  });

  it('renders a section with class `languages`', () => {
    expect(wrapper.find(`.${styles.languages}`)).toHaveLength(1);
  });

  it('renders 3 language links', () => {
    expect(wrapper.find(`.${styles.languages}`).children()).toHaveLength(3);
  });

  it('properly renders a section with class `company`', () => {
    const section = wrapper.find(`.${styles.company}`);
    expect(section).toHaveLength(1);
    expect(section.childAt(0).text()).toBe('orderbird AG Â©');
    expect(section.childAt(1).text()).toBe(`${new Date().getFullYear()}`);
  });
});
