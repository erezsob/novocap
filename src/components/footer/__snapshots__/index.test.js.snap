// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> renders properly 1`] = `
<footer
  className="footer"
>
  <div
    className="legal"
  >
    <a
      className="link"
      href=""
    >
      Terms & Conditions
    </a>
    <a
      className="link"
      href=""
    >
      Imprint
    </a>
    <a
      className="link"
      href=""
    >
      Private Policy
    </a>
    <a
      className="link"
      href=""
    >
      Pricing
    </a>
    <a
      className="link"
      href=""
    >
      Support
    </a>
    <a
      className="link"
      href="http://www.orderbird.com"
    >
      orderbird.com
    </a>
  </div>
  <div
    className="languages"
  >
    <a
      className="link currentLang"
      href=""
    >
      English
    </a>
    <a
      className="link"
      href=""
    >
      Deutsch
    </a>
    <a
      className="link"
      href=""
    >
      Français
    </a>
  </div>
  <div
    className="company"
  >
    <div>
      orderbird AG ©
    </div>
    <div>
      2018
    </div>
  </div>
</footer>
`;
